app/_src/networking/transparent-proxy/kubernetes.md

```diff
@@ -0,0 +1,265 @@
+- Annotation-based settings had to be added to each workload individually, making it difficult to manage the same configuration across multiple workloads.
+
+To address these issues, {{ Kuma }} now supports storing transparent proxy settings in dedicated ConfigMaps. These ConfigMaps can be applied at the namespace or individual workload level, offering greater flexibility and easier configuration.
+
+To enable this feature, set `{{ transparentProxy }}.configMap.enabled` during installation:
+
+{% cpinstall transparent-proxy-config-configmap-enabling %}
+{{ transparentProxy }}.configMap.enabled=true
+{% endcpinstall %}
+
+{% warning %}
+If you set `{{ transparentProxy }}.configMap.config` to an empty value, it will override `{{ transparentProxy }}.configMap.enabled` and disable the feature, even if `{{ transparentProxy }}.configMap.enabled` is set to `true`.
+{% endwarning %}
+
+<!-- vale Google.Headings = NO -->
+#### ConfigMap auto-creation and configuration
+<!-- vale Google.Headings = YES -->
+
+During installation, {{ Kuma }} will automatically create a ConfigMap in the `{{ kuma-system }}` namespace. The ConfigMap will be named based on the `{{ transparentProxy }}.configMap.name` setting, and its content will come from the YAML configuration defined in `{{ transparentProxy }}.configMap.config`, which holds the transparent proxy settings.
+
+Here is an example of how to modify parts of this configuration during installation:
+
+{% cpinstall transparent-proxy-config-configmap-auto-creation-examples %}
+{{ transparentProxy }}.configMap.config.enabled=true
+{{ transparentProxy }}.configMap.config.redirect.outbound.excludePortsForIPs=10\.1\.0\.254\,172\.10\.1.254
+{{ transparentProxy }}.configMap.config.redirect.verbose=false
+{% endcpinstall %}
+
+You can apply the same method of providing these values as explained in the **Store the desired settings in the `{{ kuma-control-plane-config }}` ConfigMap** option in the [**Modifying Control Plane Runtime Configuration**](#modifying-control-plane-runtime-configuration) section.
+
+{% tip %}
+For more information about the structure of this configuration, see the [Kuma Control Plane Helm Values]({{ docs }}/reference/kuma-cp/#helm-valuesyaml) documentation, particularly under the `{{ transparentProxy }}.configMap.config` path. You can also refer to the [Transparent Proxy Configuration Reference]({{ docs }}/reference/transparent-proxy-configuration/) for additional details.
+{% endtip %}
+
+<!-- vale Google.Headings = NO -->
+#### ConfigMap lookup strategy
+<!-- vale Google.Headings = YES -->
+
+{{ Kuma }} follows a specific order when searching for the ConfigMap:
+
+- If the workload is annotated with `traffic.kuma.io/transparent-proxy-configmap-name`, {{ Kuma }} first looks for the resource with the specified name in the workload’s namespace. If not found, it will then check the `{{ kuma-system }}` namespace for the same ConfigMap.
```

Comment:
this could be splitted to separate points to make order straightforward to read

Reply:
addressed